nix/tests/vm-tests/web-ghc/module.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, lib, config, ... }:
```

Comment:
Is this something we should have somewhere else? Do we use this in our deployment stuff? I guess not because we make it into a lambda or something, rather than as a normal systemd service on a NixOS machine?

Reply:
yeah i was thinking about that as well, but for now this module won't be used outside of this test so i'll leave it here. Should we later decide to deploy a nixos config with web-ghc as service i'll move this file out of the test folder.

Reply:
web-ghc is not in the lambda so we have a nixos definition for it. https://github.com/input-output-hk/plutus/blob/master/deployment/nixops/webghc.nix

Reply:
That could be consolidated but the current webghc.nix is not written as a nixos module and thus cannot be easily imported.

Reply:
Actually i don’t like this.. i’ll close this for now and think about what to do with this.